lib/active_interaction/base.rb

```diff
@@ -105,6 +119,9 @@ def method_missing(*args, &block)
       # @param options [Hash]
       def add_filter(klass, name, options, &block)
         raise InvalidFilterError, %("#{name}" is a reserved name) if Inputs.reserved?(name)
+        if (invalid_options = options.keys - ALLOWED_KWARGS).any?
```

Comment:
Comment on the implementation specifically. Using constant ALLOWED_KWARGS makes it impossible to extend functionality with new behavior. Consider using methods, as they can be extended.